### PR TITLE
🐛 agent rclone configuration fixes

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

Agent clone configuration appears not to be correct in master, which runs with a CEPH backend.

- ♻️ rlcone output afet sync command is more helpful (states what files it synced)
- 🐛 Fiexed issues with `MINIO` and `CEPH` in `S3Provider` enum that caused warnings in the logs. (unsure if these also blocked transfers)

## Checklist

- [x] Unit tests for the changes exist

### CODE REVIEW

File: packages/settings-library/src/settings_library/r_clone.py [hunk 1]
```diff
@@ -8,8 +8,8 @@
 
 class S3Provider(str, Enum):
     AWS = "AWS"
-    CEPH = "CEPH"
-    MINIO = "MINIO"
+    CEPH = "Ceph"
```
Comment (pcrespov) [inline packages/settings-library/src/settings_library/r_clone.py:11]: How these strings change affect/fix this sisue?  These are just identifiers, right? AFAIK could eventually be even ints

The actual information on the provider is (correctly) setup apart e.g. with this map
https://github.com/ITISFoundation/osparc-simcore/blob/cf422bb2b61b8ae738eaad79f34f1de7c2bce995/packages/settings-library/src/settings_library/utils_r_clone.py#L17

File: services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py [hunk 2]
```diff
@@ -60,9 +61,13 @@ def _get_s3_path(s3_bucket: str, labels: dict[str, str], volume_name: str) -> Pa
...
 
 
-async def _read_stream(stream):
+async def _read_stream(stream: StreamReader) -> str:
+    output = ""
     while line := await stream.readline():
-        logger.info(line.decode().strip("\n"))
+        message = line.decode()
+        output += message
+        logger.debug(message.strip("\n"))
```
Comment (sanderegg) [inline services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py:69]: Q; is it really necessary to output logs both here and in the client code? see line 132?

Comment (pcrespov) [inline services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py:69]: do you really need a log here ...  provided that you do log it as info later as well?
```python
    output = await _read_stream(process.stdout)
    await process.wait()
    logger.info(output)
```

Moreover, probably that log would "block" the advantage that you are seeking by async reading the string

File: packages/settings-library/src/settings_library/r_clone.py [hunk 3]
```diff
@@ -8,8 +8,8 @@
...
+    MINIO = "Minio"
```
Comment (sanderegg) [inline packages/settings-library/src/settings_library/r_clone.py:12]: so this is the fix?

**Review by sanderegg — APPROVED** [review]

File: services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py (hunk 2 above)
Comment (GitHK) [reply services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py:69]: I'd say yes, one is useful for debugging, if the process hangs or is taking forever there is no feedback.

The the one below in info level, is a recap at the end of the upload operation which is the only one being logged.

File: packages/settings-library/src/settings_library/r_clone.py (hunk 3 above)
Comment (GitHK) [reply packages/settings-library/src/settings_library/r_clone.py:12]: Unfortunately yes, this is the fix

File: packages/settings-library/src/settings_library/r_clone.py (hunk 1 above)
Comment (GitHK) [reply packages/settings-library/src/settings_library/r_clone.py:11]: Very good point will refactor to use these!

File: services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py (hunk 2 above)
Comment (GitHK) [reply services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py:69]: As replied above yes, to detect issues during development/debugging. If something is taking forever you cannot detect it.

File: packages/settings-library/src/settings_library/r_clone.py (hunk 3 above)
Comment (GitHK) [reply packages/settings-library/src/settings_library/r_clone.py]: Used something better now

**Review by pcrespov — APPROVED** [review]

**Pull request merged**